frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss

```diff
@@ -31,7 +31,7 @@
         background-color: inherit;
     }
 
-    .Field--error &,
+    .Field--error &:not(.ignore-error-border),
```

Comment:
could this cause issues since its a shared component across the app?

Reply:
I don't think so, as this class is not used anywhere else

Reply:
I see a lot of usages https://github.com/search?q=repo%3APostHog%2Fposthog+%3CLemonInput+&type=code
just double check its alright.